tests/bootstrap.php

```diff
@@ -0,0 +1,12 @@
+<?php
+include_once __DIR__.'/../vendor/autoload.php';
+
+$classLoader = new \Composer\Autoload\ClassLoader();
+$classLoader->addPsr4("Behat\\Mink\\Tests\\", __DIR__, true);
+$classLoader->register();
+
+// PHPUnit 6 introduced a breaking change that
```

Comment:
To address this you need to:

1. across test suite use `\PHPUnit\Framework\TestCase` (because it was available alongside with `\PHPUnit_Framework_TestCase`)
2. don't create any bootstrap file